frontend/src/component/feedbackNew/FeedbackProvider.tsx

```diff
@@ -0,0 +1,39 @@
+import { FeedbackComponent } from './FeedbackComponent';
+import { FeedbackContext } from './FeedbackContext';
+import { ReactNode, useState } from 'react';
+import { ProvideFeedbackSchema } from '../../openapi';
+
+interface IFeedbackProviderProps {
+    children: ReactNode;
+}
+
+export const DEFAULT_FEEDBACK_DATA = {
+    category: 'general',
+};
+export const FeedbackProvider = ({ children }: IFeedbackProviderProps) => {
+    const [feedbackData, setFeedbackData] = useState<ProvideFeedbackSchema>(DEFAULT_FEEDBACK_DATA);
```

Comment:
This type will change in future, and take in just feedback category and user type.